CHANGELOG.md

```diff
@@ -87,13 +87,13 @@ https://github.com/googleapis/google-cloud-cpp/issues/8234.
         `pubsub_client`, `storage_client`, `spanner_client`.
       - Some target aliases, including `bigtable::client`, `bigtable::protos`
   - **pkg-config users:** applications should use the modules starting with
-    `google_cloud_cpp`. All other modules are not retired
+    `google_cloud_cpp`. All other modules are not retired.
```

Comment:
super nit: maybe avoid the double negative here. "No other modules are retired."

Reply:
actually, it's not a double negative. I still prefer saying `¬∃ m ∈ other_modules | m.is_retired` to `∀ m ∈ other_modules | ¬m.is_retired` though

Reply:
Ack, but all I was trying to say is "now retired"